sdks/python/apache_beam/runners/interactive/interactive_beam.py

```diff
@@ -475,7 +485,9 @@ def cleanup(
             'options is deprecated since First stable release. References to '
             '<pipeline>.options will not be supported',
             category=DeprecationWarning)
-        p.options.view_as(FlinkRunnerOptions).flink_master = '[auto]'
+        p_flink_options = p.options.view_as(FlinkRunnerOptions)
+        p_flink_options.flink_master = '[auto]'
+        p_flink_options.flink_version = None
```

Comment:
Why do we reset the flink version here?

Reply:
Because Dataproc only supports a constant Flink version and we always override to that version.
We set it to None in case the user wants to use a different cluster but forgets to set the Flink version they want to use.

Beam's default value is the latest hard coded published Flink version.